frontends/web/src/routes/device/bitbox01/restore.tsx

```diff
@@ -98,9 +98,11 @@ class Restore extends Component<Props, State> {
           route('/', true);
         }
       } else {
-        alertUser(this.props.t(`backup.restore.error.e${code}`, {
-          defaultValue: errorMessage,
-        }));
+        if (typeof code === 'string') {
```

Comment:
some un-typed BitBox01 backups specific api call.

We usually know what type when we use the `src/api` backend calls, but here we call apiPost directly.